### PR TITLE
Post sidebar

### PR DESCRIPTION
**Pull request by hayden-93**
- Added custom filter utility to `eleventy.js`
- Added post sidebar to page

### CODE REVIEW

File: .eleventy.js [hunk 1]
```diff
@@ -41,6 +41,14 @@ module.exports = function (config) {
...
   config.addShortcode("dateFormat", (date, dateFormat) => format(parseISO(date), dateFormat));
   config.addShortcode("logger", (...args) => console.log(args));
+  config.addFilter("similarPosts", (collection, page, currentArticle) => {
+    const currentCategory = currentArticle.category;
+    const posts = collection.filter((post) => {
+      console.log(post.data.collections.feed);
+      return null;
+    });
+    // console.log({ page, currentArticle });
+  });
```
Comment (aron-bs) [inline .eleventy.js:51]: Can remove this now if you are not using it

Comment (hayden-93) [reply .eleventy.js]: Made changes as per the above: https://github.com/hayden-93/asos-clone/pull/33/commits/d20057748e969e1fb66593502789a2d1bf6603d7

File: pages/feed/pages.liquid [hunk 2]
```diff
@@ -1,19 +1,58 @@
...
-
-{% logger pagination.page.previous %}
 
 <section class="bg-neutral-50 py-16">
   <div class="max-w-screen-lg mx-auto px-4 grid grid-cols-12 grid-rows-2 gap-12">
     <article class="col-span-8 row-span-2 p-16 bg-neutral-400">Article</article>
-    <aside class="col-span-4 p-16 bg-neutral-400">Aside</aside>
+    <aside class="col-span-4 p-4 bg-neutral-50 space-y-4">
+      <div class="flex flex-col space-y-4">
+        <h2 class="text-center text-neutral-900 font-medium text-lg uppercase tracking-wider">Style Feed</h2>
```
Comment (aron-bs) [inline pages/feed/pages.liquid:24]: I'd probably set this as an `<h3>` or `<h4>`

Comment (hayden-93) [reply pages/feed/pages.liquid]: Made change as per the above: https://github.com/hayden-93/asos-clone/pull/33/commits/45a3e10934d748f5a5c8343bbf9de5e39f34667f

File: pages/feed/pages.liquid [hunk 3]
```diff
@@ -1,19 +1,58 @@
...
-permalink: "feed/{{ article.slug | slug }}/"
----
-
-{% logger pagination.page.previous %}
 
 <section class="bg-neutral-50 py-16">
   <div class="max-w-screen-lg mx-auto px-4 grid grid-cols-12 grid-rows-2 gap-12">
     <article class="col-span-8 row-span-2 p-16 bg-neutral-400">Article</article>
-    <aside class="col-span-4 p-16 bg-neutral-400">Aside</aside>
+    <aside class="col-span-4 p-4 bg-neutral-50 space-y-4">
```
Comment (aron-bs) [inline pages/feed/pages.liquid:22]: Does this element need any padding? If you are adding horizontal padding and gap spacing between grid columns to its parent element, does this make this additional padding unnecessary?

**Review by aron-bs — CHANGES_REQUESTED** [review]

File: pages/feed/pages.liquid (hunk 3 above)
Comment (hayden-93) [reply pages/feed/pages.liquid]: Made changes are per the above: https://github.com/hayden-93/asos-clone/pull/33/commits/53ba7896aa8b5ccba72e7451f77d7201ab5cc639

**Review by aron-bs — APPROVED** [review]

**Pull request merged**